docs/usage-and-billing.mdx

```diff
@@ -149,7 +149,8 @@ Pay through the following methods:
 
 <dl>
   <dt>Pay using your credit card.</dt>
-  <dd>The payment will be processed through Stripe.</dd>
+  <dd>The payment will be processed through Stripe. If you need to change the credit card on file, please reach out to{" "}
```

Comment:
What's the {" "} for, just curious?

Reply:
I thought it was necessary for formatting, but I just tried without and it doesn't seem to be the case.

Reply:
Never mind, it was; it seems to keep the text and the link spaced properly if there is a line break